### PR TITLE
[CIS-312] Finish SwiftUI wrappers

### PR DESCRIPTION
**Pull request by mantergo**
**In this PR:** 

- Add `SwiftUI` wrapper to `ChannelController`
- Add `SwiftUI `wrapper to `CurrentUserController`
- Group controllers related files

_WIP: Waiting for `MessageController` for `SwiftUI` wrapping_

### CODE REVIEW

File: Sources_v3/Controllers/ChannelController+SwiftUI.swift [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        @Published public private(set) var channel: ChannelModel<ExtraData>?
+        
+        ///
+        @Published public private(set) var messages: [MessageModel<ExtraData>] = []
+        
+        ///
+        @Published public private(set) var typingEvent: TypingEvent?
+        
+        ///
+        @Published public private(set) var memberEvent: MemberEvent?
```
Comment (VojtaStavik) [inline Sources_v3/Controllers/ChannelController+SwiftUI.swift]: Events can't be exposed like this. They will be accessible via Combine publishers (CIS-313)

File: Sources_v3/Controllers/CurrentUserController.swift [hunk 2]
```diff
@@ -67,7 +67,7 @@ public final class CurrentUserControllerGeneric<ExtraData: ExtraDataTypes>: Cont
     /// - Parameters:
     ///   - client: The `Client` instance this controller belongs to.
     ///   - environment: The source of internal dependencies
-    init(client: Client<ExtraData>, environment: Environment) {
+    init(client: Client<ExtraData>, environment: Environment = .init()) {
```
Comment (VojtaStavik) [inline Sources_v3/Controllers/CurrentUserController.swift:70]: 👍

**Review by VojtaStavik — APPROVED** [review]
Perfect, thank you! Good idea with the grouping, it was becoming quite busy.

**Comment by VojtaStavik** [comment]
FYI @cardoso , more SwiftUI wrappers

File: Sources_v3/Controllers/ChannelController+SwiftUI.swift (hunk 1 above)
Comment (mantergo) [reply Sources_v3/Controllers/ChannelController+SwiftUI.swift]: I see, so should I just remove them from `SwiftUI `wrapper?

Comment (VojtaStavik) [reply Sources_v3/Controllers/ChannelController+SwiftUI.swift]: Yep :)

**Pull request merged**